src/components/interfaces.ts

```diff
@@ -23,26 +23,28 @@
  SOFTWARE.
 
  */
-
 import { ILookmlModelExploreField } from "@looker/sdk"
 
 export interface ColumnDescriptor {
   name: string
   label: string
   rowValueDescriptor: string
```

Comment:
`rowValueDescriptor: keyof ILookmlModelExploreField`?

Reply:
This would work, but there is a column formatter for the Comments functionality that uses a "comment" `rowValueDescriptor`... A key not present on an ExploreField

Reply:
`keyof ILookmlModelExploreField | 'comment'`?